tests/platform_tests/sfp/test_show_intf_xcvr.py

```diff
@@ -57,3 +58,26 @@ def test_check_sfpshow_eeprom(duthosts, enum_rand_one_per_hwsku_frontend_hostnam
         if intf not in xcvr_skip_list[duthost.hostname]:
             assert intf in parsed_eeprom, "Interface is not in output of 'sfputil show eeprom'"
             assert parsed_eeprom[intf] == "SFP EEPROM detected"
+
+
+def test_check_show_lpmode(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    """
+    @summary: verify port mode in  'show interface transceiver lpmode'
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
```

Comment:
also need to for if the interface is not in the xcvr skip list and then check for the valid lpmode output